core/vm/contracts.go

```diff
@@ -645,6 +648,8 @@ func (c *getValidator) Run(input []byte, caller common.Address, evm *EVM, gas ui
 
 	index := (&big.Int{}).SetBytes(input[0:32])
 
+	validators := evm.Context.Engine.GetValidators(big.NewInt(int64(evm.Context.BlockNumber.Uint64()-1)), evm.Context.GetHash(evm.Context.BlockNumber.Uint64()-1))
```

Comment:
You might use `BlockNumber.Int64()` instead to avoid the  extra conversion to `int64`